packages/arb-validator/rollup/node.go

```diff
@@ -311,6 +328,21 @@ func GeneratePathProof(from, to *Node) []common.Hash {
 	return append(sub, to.innerHash)
 }
 
+func IsDownstream(from, to *Node) []common.Hash {
+	// returns nil if no proof exists
+	if to == nil {
+		return nil
+	}
+	if from == to {
+		return []common.Hash{}
+	}
+	sub := GeneratePathProof(from, to.prev)
+	if sub == nil {
+		return nil
+	}
+	return append(sub, to.innerHash)
+}
+
```

Comment:
This looks very similar to GeneratePathProof.  Suggest adding a comment explaining the difference.

Reply:
This was unused so I'm just killing it. I don't remember why it was originally included